File: 1-11-2021/dash.php
<?php
ob_start();
session_start();
if($_SESSION['email']==''){
    header('Location:login.php');
}
else{

?>

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/style.css">
    <title>Dashboard</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    <style>
        /* logout button */

          .logout input{
            position:absolute;
            right: 10px;
            top:8px;
          } 
        /* table */
        .table {
    --bs-table-bg: transparent;
    --bs-table-accent-bg: transparent;
    --bs-table-striped-color: #212529;
    --bs-table-striped-bg: rgba(0, 0, 0, 0.05);
    --bs-table-active-color: #212529;
    --bs-table-active-bg: rgba(0, 0, 0, 0.1);
    --bs-table-hover-color: #212529;
    --bs-table-hover-bg: rgba(0, 0, 0, 0.075);
    width: 76%;
    margin-bottom: 1rem;
    color: #212529;
    vertical-align: top;
    border-color: #dee2e6;
    margin-left:10%;
    margin-top:50px;
}

    </style>
</head>
<body>
<div class="top">  
      <div class="logout">
          <form method='POST'>
          <input type="submit" value="Logout" name="logout" class="btn btn-primary">
          </form>
          </div>
          </div>
        
</body>
</html>
<?php 
    include 'main.php';

    $semail = $_SESSION['email'];
    $query = mysqli_query($conn,"select * from Students") or die("Error in query");
    
    $sql ="select Name from Students where Email ='$semail'";
    
    $q1 = mysqli_query($conn,$sql); 
    $r = mysqli_fetch_array($q1);
    
    echo "<center><b><h3>Welcome ".$r["Name"]."</center>";
    echo"<table class='table'>
    <thead>
      <tr>
        <th scope='col'>ID</th>
        <th scope='col'>Name</th>
        <th scope='col'>Mobile</th>
        <th scope='col'>Email</th>
        <th scope='col'>Password</th>
        <th scope='col'>Address</th>
        <th scope='col'>Edit</th>
        <th scope='col'>Delete</th>
      </tr>
    </thead>";
    // echo"<center><table border=2><tr><th>Id</th><th>Name</th><th>Mobile</th><th>Email</th><th>Password</th><th>Address</th><th>Edit</th><th>Delete</th></tr>";
    while($row=mysqli_fetch_array($query))
    {
        $s_id=$row['Id'];
        echo "<tr>";
        echo "<td>".$row['Id']."</td>";
        echo "<td>".$row['Name']."</td>";
        echo "<td>".$row['Mobile']."</td>";
        echo "<td>".$row['Email']."</td>";
        echo "<td>".$row['Password']."</td>";
        echo "<td>".$row['Address']."</td>";
        
        echo "<td><a href='update.php?s_id=$s_id'>Edit</a></td>";
        echo "<td><a href='Delete.php?s_id=$s_id'>Delete</a>.</td>";
    
    }       
    echo "</table></center>";
    }
    mysqli_close($conn);
?>
<?php
if(isset($_POST['logout'])){
    header('Location:logout.php');
}
?>

File: 08-dec-2021/dashborad/Dash.py
#!/usr/bin/python3
import pymysql
import cgi, cgitb

from pymysql import cursors
cgitb.enable()
form = cgi.FieldStorage()
db = pymysql.connect(host='localhost',user='root',password='',database='pypractice')
cursor=db.cursor()

cursor.execute("SELECT*from pythonn")
data=cursor.fetchall()

print("Content-type:text/html\r\n\r\n")

print("<html>")
print("<head>")
print('<link rel="stylesheet" href="style.css">')
print("<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>")
print("<link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css' integrity='sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3' crossorigin='anonymous'>")
print('<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">')
print("<title> Dashboard </title>")
print('<style> a{text-decoration: none; color:black} </style')
print("</head>")
print ("<body>")
print('<div class="container">')
print('<div class="btn btn-danger" style="width:100px"> <a href="loginn.html"> Log-Out</a> </div>')


print('<center> <h1>Student Information </h1> </center>')
print("<center> <table class='table table-striped'>")
print("<thead>")
print("<tr>")
print("<th scope='col'>ID</th>")
print("<th scope='col'>Name</th>")
print("<th scope='col'>Address</th>")
print("<th scope='col'>Mobile</th>")
print("<th scope='col'>Email</th>")
print("<th scope='col'>Action</th>")
print("</tr>")
print("</thead>")      
print("<tbody>")

for row in data:
    id=row[0]
    name=row[1]
    address=row[2]
    mobile=row[3]
    email=row[4]
    print("<tr> <td>%s</td> "%(id))
    print("<td>%s</td>"%(name))
    print("<td>%s</td>"%(address))
    print("<td>%s</td>"%(mobile))
    print("<td>%s</td>"%(email))
    print('<td><a href="update.py?id=%s"> <i class="fa fa-edit" style="font-size:25px;color:blue"></i> '%(id))
    print('<a href="delete.py?id=%s"><i class="fa fa-trash-o" style="font-size:25px;color:red"></i></td>'%(id))
    
print("</tbody> </table> </ceter>")

print('</div>')
    
print ("</body>")
print ("</html>")
# disconnect from serve
db.close()
